nbgitpuller/handlers.py

```diff
@@ -65,16 +100,31 @@ def get(self):
             # so that all repos are always in scope after cloning. Sometimes
             # server_root_dir will include things like `~` and so the path
             # must be expanded.
-            repo_parent_dir = os.path.join(os.path.expanduser(self.settings['server_root_dir']),
-                                           os.getenv('NBGITPULLER_PARENTPATH', ''))
-            repo_dir = os.path.join(repo_parent_dir, self.get_argument('targetpath', repo.split('/')[-1]))
+            repo_parent_dir = os.path.join(os.path.expanduser(self.settings['server_root_dir']), os.getenv('NBGITPULLER_PARENTPATH', ''))
+            nbgitpuller.REPO_PARENT_DIR = repo_parent_dir
+
+            repo_dir = os.path.join(
+                        repo_parent_dir,
+                        self.get_argument('targetpath', repo.split('/')[-1]))
```

Comment:
what is the standard on the number of columns(line length) -- I was getting angst about long line lengths!

Reply:
We have been using `black` autoformatter to handle these things automatically for us in the JupyterHub GitHub org repos, typically black gets it below 88. This repo isn't setup to use `black` yet though.

I suggest try to relax knowing it will be autoformatted in the end! :D

Reply:
got it!